FpsCheck.py

```diff
@@ -35,6 +35,7 @@ class FpsCheck:
                 without any given check failing halfway through and not 
                 catching other issues. 
   """
+    WELLKNOWN = "/.well-known/first-party-set.json"
```

Comment:
```suggestion
    WELL_KNOWN = "/.well-known/first-party-set.json"
```
Nit: Python typically names things in CONSTANT_CASE using underscores to separate words. See https://peps.python.org/pep-0008/#constants